pkg/controller/user/user_controller.go

```diff
@@ -265,6 +294,143 @@ func (c *Controller) encryptPassword(user *iamv1alpha2.User) (*iamv1alpha2.User,
 	return user, nil
 }
 
+func (c *Controller) ensureNotControlledByKubefed(user *iamv1alpha2.User) error {
+	if user.Labels[constants.KubefedManagedLabel] != "false" {
+		if user.Labels == nil {
+			user.Labels = make(map[string]string, 0)
+		}
+		user = user.DeepCopy()
+		user.Labels[constants.KubefedManagedLabel] = "false"
+		_, err := c.ksClient.IamV1alpha2().Users().Update(user)
+		if err != nil {
+			klog.Error(err)
+		}
+	}
+	return nil
+}
+
+func (c *Controller) multiClusterSync(user *iamv1alpha2.User) error {
+
+	if err := c.ensureNotControlledByKubefed(user); err != nil {
+		klog.Error(err)
+		return err
+	}
+
+	obj, exist, err := c.fedUserCache.GetByKey(user.Name)
+
```

Comment:
don't need blank line, put error checking code right after statement is more clear